server/odc-core/src/main/java/com/oceanbase/odc/core/sql/execute/model/SqlExecTime.java

```diff
@@ -33,5 +33,7 @@ public class SqlExecTime {
 
     // 链路信息
     private TraceSpan traceSpan;
+    private boolean withFullLinkTrace;
+    private String traceEmptyReason;
```

Comment:
traceEmptyReason  or emptyTraceReaon, which will be better?

Reply:
I prefer traceEmptyReason. The other seems like a boolean variable that indicates the 'trace reason' is empty.